tests/Vue/Components/TextArea.spec.js

```diff
@@ -22,6 +22,14 @@ describe('TextArea.vue', () => {
         expect(wrapper.find('textarea').classes()).toContain('wikit-TextArea__textarea--horizontal');
     });
 
+    it('ignores invalid resize values', () => {
+        const wrapper = mount(TextArea, {
+            propsData: { resize: 'nonsense' }
+        });
+
+        expect(wrapper.find('textarea').classes()).not.toContain('wikit-TextArea__textarea--nonsense');
```

Comment:
Good workaround! :D and much simpler

Reply:
I think I disagree and this should be:
```suggestion
        expect(wrapper.find('textarea').classes()).toContain('wikit-TextArea__textarea--vertical');
```
because that's what we put in the class list for textarea for empty or nonsense `resize` values.